after-your-talk.txt

```diff
@@ -52,3 +52,7 @@ During Q&A you may have someone point out that you said something inaccurate,
 or missed pointing out something useful. Accept this sort of feedback happily.
 If you made a mistake, admit to it immediately. Being totally honest will win
 you more credibility than pretending to be right all the time.
+
+4. Admit what you don't know.
+
+If you're not sure, say so. Others seeing that you knowing that you are wrong is better than it seeming like you do not know you are wrong. It's better than having someone disprove you. In this case, say that you will get back to them, get some sort of contact information, and compliment their question. After admitting your uncertainty, you are certainly welcome to make a guess, and provide your rationale for your thinking.
```

Comment:
@elizabethsiegle I recommend: "Others seeing that you know<strike>ing</strike> that you are wrong"